src/mods/common/analyzer/update/bpy.types.mod.rst

```diff
@@ -47,6 +47,45 @@
       :type seq: collections.abc.Sequence[bool] | collections.abc.Sequence[int] | collections.abc.Sequence[float] | typing_extensions.Buffer
       :mod-option arg seq: skip-refine
 
+.. class:: bpy_prop_collection_idprop
+
+   :generic-types: _GenericType1
+
+   .. method:: values()
+
+      :rtype: list[_GenericType1 | None]
+      :mod-option rtype: skip-refine
+
+   .. method:: find(key)
+
+      :type key: str (never None)
```

Comment:
I think return value is required.

Reply:
The return type seems correct (i.e. `int`).
![image](https://github.com/user-attachments/assets/4dbaa78e-59ab-4233-b103-8b09559ed53e)

Reply:
Thank you for the confirmation.
I thought this was a append mod file.
Looks good to me.